src/_stylesheets/_section-highlight.scss

```diff
@@ -50,3 +56,7 @@
 .app-section-highlight__wrapper--space-around {
   margin: govuk-spacing(8);
 }
+
+.app-section-highlight__size-recommendation {
+  @include govuk-font($size: 24, $weight: bold);
+}
```

Comment:
Ponder: As these styles are only used twice on the site, both on the same page, could they be replaced outright with Frontend override classes or page-specific CSS?